src/05-led-roulette/build-it.md

```diff
@@ -45,9 +45,9 @@ $ cargo build --target thumbv7em-none-eabihf
     Finished dev [unoptimized + debuginfo] target(s) in 35.84 secs
 ```
 
-> **NOTE** Be sure to compile this crate *without* optimizations
+> **NOTE** Be sure to compile this crate *without* optimizations. The provided Cargo.toml file used with the build command givn above will ensure optimizations are off. Just don't add any extra build options.
```

Comment:
There's a typo in there "givn". However I'd simplify this note by simply joining the existing sentence with your last one. The important part is that the default compile target is `dev` which is what we want here.